llvm/include/llvm-c/Core.h

```diff
@@ -3682,6 +3682,33 @@ LLVMValueRef LLVMInstructionClone(LLVMValueRef Inst);
  */
 LLVMValueRef LLVMIsATerminatorInst(LLVMValueRef Inst);
 
+/**
+ * Obtain the first debug record attached to an instruction.
+ *
+ * The returned debug record can be used as an iterator. You will likely
+ * eventually call into LLVMGetNextDbgRecord() with it.
```

Comment:
We should mention these return null. Something like: "Returns the first DbgRecord attached to Inst or NULL if there are none [...]".

Same thing applies to the other new functions ones below.

Reply:
Done.

I also changed "Returns" to "Return", to make it consistent with the other sentences, which are imperative.